domains/cutyoururl.ashishagarwal.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "CutYourURL is a free URL Shortener!",
+  "repo": "https://github.com/ashishagarwal203/CutYourURL",
+  "owner": {
+    "username": "ashishagarwal2023",
+    "twitter": "",
+    "email": "code.with.aasheesh@gmail.com"
```

Comment:
if you dont want to list your twitter then remove the twitter line

Reply:
ok